Automattic/wp-calypso
# Fix stylelint errors in scss files

#### Changes proposed in this Pull Request

* Fix some `stylelint` errors manually
* Run `stylelint --fix`
* Disable `selector-max-id` rule in few files where IDs were used extensively (mostly stuff like Calypso's `#secondary`/`#footer` or TinyMCE IDs)

#### Testing instructions

- Run `npm run lint:css` and confirm no more errors
- Calypso still looks great :-)

#### Related
- lint all scss files https://github.com/Automattic/wp-calypso/pull/32350
- reformat scss files https://github.com/Automattic/wp-calypso/pull/32351
